js/homework.js

```diff
@@ -0,0 +1,84 @@
+var bluChange = document.querySelector('#blueBtn');
+// select the button and assign to the bluChange variable
+bluChange.addEventListener('click', changeBlueColor);
+// create an Event and call the function
+function changeBlueColor() {
+    var myJumbo = document.querySelector('.jumbotron');
+    var myDonate = document.querySelector('.btn-primary')
+    var volnter = document.querySelector('.btn-secondary')
+    myJumbo.style.backgroundColor = '#588fbd';
+    myDonate.style.backgroundColor = '#ffa500';
+    volnter.style.backgroundColor = 'black';
+    volnter.style.color = 'white';
+}
+
+var oraChange = document.querySelector('#orangeBtn');
+// select the button and assign to the oraChange variable
+oraChange.addEventListener('click', changeOrangeColor);
+// create an Event and call the function
+function changeOrangeColor() {
+    //change the color of selected query
+    var myJumbo = document.querySelector('.jumbotron');
```

Comment:
If you are defining the same selectors as variables in your js file, put them once at the top of the file. 

As long as they got the var at the front they will have a local scope to that "file". That way you can:
* Reduce the complexity of your code by reducing the number of lines
* Avoid needing to change multiple lines when a selector changes

That also produces other benefits like ease to read what, as a side effect, make for bugs to be easier to be found and you to be happier ;)